src/app/app-routing.module.ts

```diff
@@ -1,22 +1,19 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule, Routes } from '@angular/router';
-import { CharacterDetailsComponent } from './components/character-details/character-details.component';
 import { SpeciesListComponent } from './components/species-list/species-list.component';
-import { CharacterListComponent } from './components/character-list/character-list.component';
 import { PlanetsListComponent } from './components/planets-list/planets-list.component';
 
 
 export const routes: Routes = [
-  {
-    path: 'people/:page',
-    component: CharacterListComponent
-  },
-  {
+    {
+      path: 'people',
+      loadChildren: () => import('src/app/modules/character.module').then(mod => mod.CharacterModule),
```

Comment:
Help me understand why we're taking this approach by explaining the problem you're trying to solve.

Reply:
We created a separate module for the character routing, since all of our routes fall under '/people'. Just thought it was modular to have a separate module instead of updating the app-routing module constantly.

Reply:
help to make it modular like our new character module to keep it separate and prevent merge conflicts

Reply:
Also since we are creating a separate module, we believe that it would be more organized by grouping the routes that are related than just dumping all the routes into the app-routing.module.ts

Reply:
I see what you mean since I think you're trying to implement nested routes. I think I missed the routes in the module. Nice work.